lib/rubocop/formatter/disabled_config_formatter.rb

```diff
@@ -23,6 +23,12 @@ class << self
         attr_accessor :config_to_allow_offenses
       end
 
+      def file_started(_file, file_info)
+        @exclude_limit_option =
+          file_info.fetch(:cli_options, {})[:exclude_limit]
+        @exclude_limit = (@exclude_limit_option || 15).to_i
```

Comment:
Better to keep `MAXIMUM_EXCLUSION_ITEMS` around but call it `DEFAULT_MAXIMUM_EXCLUSION_ITEMS`, I think.